R/image_converter.R

```diff
@@ -0,0 +1,114 @@
+#' @title Image Converter
+#'
+#' @description
+#' A utility to load an image or set of images into a single dataset that can
+#' then be used by other mlpack methods and utilities. This can also unpack an
+#' image dataset into individual files, for instance after mlpack methods have
+#' been used.
+#'
+#' @param input Image filenames which have to be loaded/saved.
```

Comment:
Should we print in the documentation here what the type of each parameter is?  This one is a vector of strings, but I can't see that just from reading this documentation.  I'm not sure if R provides facilities for automatically printing the expected type of each input parameter (probably not?).

Reply:
@rcurtin unfortunately, that isn't possible as _R_'s type system is weak. There is no `int x` available. The best that can be done is probably inferring it from the _C++_ bindings and then saying,

- If `int`, then a single `integer` value, 
- If `arma::vec`, then a vector of `numeric` values, 
- ...

Reply:
Yeah---we can definitely do this from C++ then.  All we need to do is associate some documentation string with each C++ type (just like you wrote above in the short list :)) and then we just need to print the associated documentation for each type.  @Yashwants19 do you think you could add something like that?  Let me know if I can clarify anything.  Or if you have a better idea for how to accomplish the same goal go ahead. :+1:

Reply:
I am not sure. Do we want this?
```
#' @param input Character Vector - Image filenames which have to be loaded/saved.
```

Reply:
Maybe it would be sufficient to do this?
```
#' @param input Image filenames which have to be loaded/saved (character vector).
```
But in any case I think something like this is probably the best approach.

Reply:
Great, I will made the respective changes :+1:.